apps/frontend/src/pages/project/project.tsx

```diff
@@ -1,43 +1,77 @@
 import {
 	Breadcrumbs,
 	Button,
+	Modal,
 	PageLayout,
 } from "~/libs/components/components.js";
 import { AppRoute, DataStatus } from "~/libs/enums/enums.js";
 import {
 	useAppDispatch,
 	useAppSelector,
+	useCallback,
 	useEffect,
 	useModal,
 	useParams,
 } from "~/libs/hooks/hooks.js";
-import { actions as projectActions } from "~/modules/projects/projects.js";
+import {
+	actions as projectActions,
+	type ProjectPatchRequestDto,
+} from "~/modules/projects/projects.js";
 import { NotFound } from "~/pages/not-found/not-found.jsx";
+import { ProjectUpdateForm } from "~/pages/projects/libs/components/components.js";
 
-import { SetupAnalyticsModal } from "./libs/components/components.js";
+import {
+	ProjectMenu,
+	SetupAnalyticsModal,
+} from "./libs/components/components.js";
 import styles from "./styles.module.css";
 
 const Project = (): JSX.Element => {
+	const handleManageAccess = useCallback(() => {}, []);
```

Comment:
Perhaps it would be better to either remove it and make `onClick` optional, or to navigate using the function?

Reply:
I’ve made onClick optional, so now we can use it when needed or rely on NavLink for navigation. It keeps things flexible.